apps/crn-server/src/data-providers/users.data-provider.ts

```diff
@@ -347,6 +355,24 @@ export const parseGraphQLUserToDataObject = (
       : 'Guest';
   const teams = parseGraphQLUserTeamConnections(flatTeams || []);
 
+  const workingGroups = item.referencingWorkingGroupsContents
+    ? item.referencingWorkingGroupsContents.map((wg) => {
+        const wgRole = wg.flatData.leaders?.reduce((result, leader) => {
+          if (leader.user?.[0]?.id === item.id) {
+            return leader.workstreamRole || 'Member';
+          }
+          return result;
+        }, 'Member');
+
+        return {
+          id: wg.id,
+          name: wg.flatData.title || '',
+          role: wgRole || '',
```

Comment:
Maybe we don't need this `|| ''` because working group `title` is required and the role is `Member` by default in the reduce above 😉